src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -106,27 +108,27 @@ export const styles = StyleSheet.create({
   containerPhoto: {
     justifyContent: 'center',
     alignItems: 'center',
-    paddingTop: 8,
-    paddingBottom: 13
+    paddingTop: dimen.normalizeDimen(8),
+    paddingBottom: dimen.normalizeDimen(13)
   },
   btnUpdatePhoto: {
-    width: normalize(100),
-    height: normalize(100),
-    borderRadius: normalize(50),
+    width: dimen.normalizeDimen(100),
+    height: dimen.normalizeDimen(100),
+    borderRadius: dimen.normalizeDimen(50),
     backgroundColor: COLORS.lightgrey,
     justifyContent: 'center',
     alignItems: 'center'
   },
   groupProfilePicture: {
-    width: normalize(100),
-    height: normalize(100),
-    borderRadius: normalize(50),
-    paddingLeft: 8
+    width: dimen.normalizeDimen(100),
+    height: dimen.normalizeDimen(100),
+    borderRadius: dimen.normalizeDimen(50),
+    paddingLeft: dimen.normalizeDimen(8)
   },
   loading: {
     justifyContent: 'center',
     alignItems: 'center',
-    marginBottom: normalize(20)
+    marginBottom: dimen.normalizeDimen(20)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The normalization function `dimen.normalizeDimen` is used instead of `normalize` for styling. This change improves consistency in the codebase by using the same normalization function throughout.
```diff
- paddingTop: 8,
- paddingBottom: 13
+ paddingTop: dimen.normalizeDimen(8),
+ paddingBottom: dimen.normalizeDimen(13)

- width: normalize(100),
- height: normalize(100),
- borderRadius: normalize(50),
+ width: dimen.normalizeDimen(100),
+ height: dimen.normalizeDimen(100),
+ borderRadius: dimen.normalizeDimen(50),

- marginBottom: normalize(20)
+ marginBottom: dimen.normalizeDimen(20)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->